wp-cli-templates/block-index-js.mustache

```diff
@@ -30,24 +28,23 @@ export const settings = {
 		</Fragment>
 	),
 
-	/* @TODO add icon */
+	/* @TODO Add the icon. You can use one of these https://material.io/tools/icons/?style=outline */
 	icon: renderMaterialIcon(
 		<Path d="M9 15h2V9H9v6zm1-10c-.5 0-1 .5-1 1s.5 1 1 1 1-.5 1-1-.5-1-1-1zm0-4c-5 0-9 4-9 9s4 9 9 9 9-4 9-9-4-9-9-9zm0 16c-3.9 0-7-3.1-7-7s3.1-7 7-7 7 3.1 7 7-3.1 7-7 7z" />
 	),
 
 	category: 'jetpack',
 
-	/* @TODO add keywords */
-	keywords: [ {{#keywords}}__( '{{ keyword }}', 'jetpack' ), {{/keywords}} ],
+	/* @TODO Add up to three keywords */
+	keywords: [ {{#keywords}}_x( '{{ keyword }}', 'block search term', 'jetpack' ), {{/keywords}} ],
 
 	supports: {
 		html: false,
 	},
 
-	/* @TODO edit */
-	{{#externalEdit}}edit,{{/externalEdit}}
-	{{^externalEdit}}edit: () => <div>Hi! The edit view for this block goes here.</div>,{{/externalEdit}}
+	/* @TODO Write the block editor UI */
+	edit: () => <div>Hi! The edit view for this block goes here.</div>,
```

Comment:
did you mean to leave this as `edit,` since the import is still on top of the file?

Reply:
😅